packages/opencensus-core/src/trace/model/tracer.ts

```diff
@@ -40,6 +38,7 @@ export class CoreTracer implements types.Tracer {
   /** A list of end span event listeners */
   private eventListenersLocal: types.SpanEventListener[] = [];
   /** A list of ended root spans */
+  // @ts-ignore
```

Comment:
Why we disabling if we want to avoid redundant definitions?

Reply:
As discussed, I added to suppress errors. I am not completely aware of CoreTracer properties, hence didnt remove that variable. Anyways, I will remove it in next PR.

Reply:
> As discussed, I added to suppress errors. I am not completely aware of CoreTracer properties, hence didnt remove that variable. Anyways, I will remove it in next PR.

How users will use Typescript for their projects in this case?

Reply:
Looks like each case it's a private unused variable, so we should clean them up. Let's just remove them in the next PR like @mayurkale22 suggested. They aren't api breaking to remove since they're private.